docs/using-the-compiler.rst

```diff
@@ -289,6 +289,14 @@ Input Description
               "stackAllocation": true,
               // Select optimization steps to be applied.
               // Optional, the optimizer will use the default sequence if omitted.
+              // Select optimization steps to be applied. It is also possible to modify both the
+              // optimization sequence and the cleanup sequence. Instructions for each sequence
+              // are separated with the : delimiter. For more information see
+              // "The Optimizer > Selecting Optimizations".
+              // Optional, the optimizer will use the default sequence if omitted. Omitting the
```

Comment:
Starting with `Optional, ...` reads a bit weird now that the above sentence has been expanded. Perhaps something like `This field is optional, and if not provided, the optimizer will ...`